packages/openapi-ts/src/utils/write/types.ts

```diff
@@ -184,6 +186,116 @@ const processType = (client: Client, model: Model, onNode: OnNode) => {
     onNode,
     type: toType(model),
   });
+
```

Comment:
**These changes are the main meat of this PR**, iterate through a model and any refs on that model, and generate transformer functions which deeply mutate a given input through object properties and arrays.